<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deliver_form</class>
 <widget class="QWidget" name="deliver_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Приходная накладная</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="bill_number_date">
       <property name="text">
        <string>Накладная №1 от 10.10.10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Поставщик</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="save_close_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Провести накладную</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="save_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить и не проводить</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Всего лс на сумму 0.00р</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Дата поступления</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="deliver_date">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>20</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>00.00.0000; </string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="close_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>500</height>
        </size>
       </property>
       <property name="text">
        <string>Закрыть без сохранения</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="9">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="distributor_combo"/>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="print_cards_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Печать стел. карт</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="print_bill_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Печать накладной</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Источник финансирования</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6" colspan="2">
      <widget class="QComboBox" name="financing_combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Источник финансирования</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
